src/velocitas/model_generator/cpp/cpp_generator.py

```diff
@@ -65,6 +67,31 @@ def generate(self):
 
         self.__gen_model(self.root_node, self.root_namespace_list, is_root=True)
         self.__visit_nodes(self.root_node, self.root_namespace_list)
+        # self.__gen_cmake_project()
+        self.__gen_conan_package()
+
+    def __gen_conan_package(self):
+        path = os.path.join(self.target_folder, "conanfile.py")
+        with open(path, "w", encoding="utf-8") as file:
+            file.write(
+                """from conans import ConanFile
+
+class VehicleModelConan(ConanFile):
+    name = "vehicle-model"
+    version = "generated"
+    license = "Apache 2.0"
+    url = "https://github.com/eclipse-velocitas/vehicle-model-generator"
+    description = "Vehicle Model API auto-generated from Vehicle Signal Specification"
```

Comment:
This should be reworked later to contain meaningful values

Reply:
We will do that later, as discussed.